liquidhaskell-boot/src/Language/Haskell/Liquid/Types/Types.hs

```diff
@@ -261,28 +263,40 @@ instance Monoid LogicMap where
 instance Semigroup LogicMap where
   LM x1 x2 <> LM y1 y2 = LM (M.union x1 y1) (M.union x2 y2)
 
-data LMap = LMap
+data LMapV v = LMapV
```

Comment:
To minimize changes, I've kept the name of the data constructor as it was for other types.
```
data LMapV v = LMap
  { ...
  }
```